start_esp/nginx-auto.conf.template

```diff
@@ -75,6 +75,26 @@ http {
   server {
     server_name ${ingress.host};
 
+% if experimental_enable_security_config_from_cipherli_st:
+    # The Nginx security configuration recommended on https://cipherli.st/
+    ssl_protocols TLSv1.3; # Requires nginx >= 1.13.0 else use TLSv1.2
+    ssl_prefer_server_ciphers on;
+    ssl_dhparam /etc/nginx/dhparam.pem; # openssl dhparam -out /etc/nginx/dhparam.pem 4096
```

Comment:
this may fail since we dont have this file

Reply:
dhparam.pem should be passed or generated at entrypoing